src/main/java/com/karankumar/bookproject/backend/entity/RatingScale.java

```diff
@@ -26,7 +26,7 @@
 
 @Log
 public enum RatingScale {
-    NO_RATING("No rating", null),
```

Comment:
Sorry, could you revert this? We want "No rating" as the string. "No" by itself is not descriptive. This may mean that you need to add an if condition to see whether the rating is set to `NO_RATING`. If so, then you can remove the duplicated word "rating" from the string "No rating rating"